solidity/contracts/utility/interfaces/ITokenHolder.sol

```diff
@@ -9,9 +9,17 @@ import "./IOwned.sol";
     Token Holder interface
 */
 interface ITokenHolder is IOwned {
+    receive() external payable;
+
     function withdrawTokens(
-        IERC20 _token,
-        address _to,
-        uint256 _amount
+        IERC20 token,
+        address to,
+        uint256 amount
+    ) external;
+
+    function withdrawMultipleTokens(
```

Comment:
maybe `withdrawTokensMultiple` is better in this case? not sure... :)

Reply:
👍

Reply:
Done